nidn/fdtd_integration/compute_spectrum_fdtd.py

```diff
@@ -33,11 +34,11 @@ def compute_spectrum_fdtd(permittivity, cfg: DotMap):
 
     if (len(cfg.PER_LAYER_THICKNESS) == 1) and (cfg.N_layers > 1):
         cfg.PER_LAYER_THICKNESS = cfg.PER_LAYER_THICKNESS * cfg.N_layers
-    thicknesses = torch.tensor(cfg.PER_LAYER_THICKNESS)
+    cfg.thicknesses = torch.tensor(cfg.PER_LAYER_THICKNESS)
     # The scaling is the number of grid points per unit magnitude. This is the maximum of the relation between the unit magnitude and 1/10th of the smallest wavelength,
     # and a constant which is defaulted to 10. If this scaling becomes too low, i.e. below 2, there might be some errors in creating the grid,
     # as there are too few grid points for certain elements to be placed correctly.
-    scaling = torch.maximum(
+    cfg.scaling = torch.maximum(
```

Comment:
we might want to give it a more verbose name, maybe cfg.FDTD_grid_scaling ? otherwise, if I read the cfg later I have no idea which scaling this refers to

Reply:
Good point!